dev/plugins/hu.elte.txtuml.utils.eclipse/src/hu/elte/txtuml/utils/jdt/ElementTypeTeller.java

```diff
@@ -402,4 +408,8 @@ public static boolean isDelegation(ITypeBinding binding) {
 		return SharedUtils.typeIsAssignableFrom(binding, Delegation.class);
 	}
 
+	public static boolean isModelEnum(ITypeBinding binding) {
```

Comment:
Once methods for declarations and methods for bindings were separated in this file, but as I see, this rule is violated now, so I change.